app/controllers/application_controller.rb

```diff
@@ -1,10 +1,15 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :create_cart
```

Comment:
We don't see any problem with this, both ways make sense to us so we will keep it the way it is. Let's move on to the next feature.